src/libsyntax/diagnostic_list.rs

```diff
@@ -16,31 +16,394 @@
 register_long_diagnostics! {
 
 E0533: r##"
+The export_name attribute was malformed.
+
+Erroneous code example:
+
 ```compile_fail,E0533
-#[export_name]
+#[export_name] // error: export_name attribute has invalid format
+pub fn something() {}
+
+fn main() {}
+```
+
+The export_name attribute expects a string in order to determine the name of
+the exported symbol. Example:
+
+```
+#[export_name = "some_function"] // ok!
+pub fn something() {}
+
+fn main() {}
+```
+"##,
+
+E0534: r##"
+The inline attribute was malformed.
+
+Erroneous code example:
+
+```compile_fail,E0534
+#[inline()] // error: expected one argument
+pub fn something() {}
+
+fn main() {}
+```
+
+The parenthesized inline attribute requires the parameter to be specified:
+
+```ignore
+#[inline(always)]
+fn something() {}
+
+// or:
+
+#[inline(never)]
+fn something() {}
+```
+
+Alternatively, a paren-less version of the attribute may be used to hint the
+compiler about inlining opportunity:
+
+```
+#[inline]
+fn something() {}
+```
+
+For more information about the inline attribute, read:
+https://doc.rust-lang.org/reference.html#inline-attributes
+"##,
+
+E0535: r##"
+An unknown argument was given to inline attribute.
+
+Erroneous code example:
+
+```compile_fail,E0535
+#[inline(unknown)] // error: invalid argument
 pub fn something() {}
 
 fn main() {}
 ```
+
+The inline attribute only supports two arguments:
+
+ * always
+ * never
+
+All other arguments given to the inline attribute will return this error.
+Example:
+
+```
+#[inline(never)] // ok!
+pub fn something() {}
+
+fn main() {}
+```
+
+For more information about the inline attribute, https:
+read://doc.rust-lang.org/reference.html#inline-attributes
+"##,
+
+E0536: r##"
+No cfg-pattern was found for `not` statement.
```

Comment:
Also this error happens when multiple patters are provided to the predicate as well, thus making this short description somewhat invalid.

I would instead write this as “The `not` cfg-predicate was malformed.” or something.